packages/design-system/src/components/FormLabel/FormLabel.test.jsx

```diff
@@ -101,8 +101,8 @@ describe('FormLabel', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
-  it('supports additional classNames', () => {
-    const props = { className: 'ds-u-foo', labelClassName: 'ds-u-bar' };
+  it('supports additional attributes', () => {
+    const props = { className: 'ds-u-foo', ariaLabel: 'testing aria' };
     const wrapper = shallow(<FormLabel {...props}>{labelText}</FormLabel>);
 
```

Comment:
Maybe add the check for the ariaLabel here?
`expect(wrapper.prop('ariaLabel')).toBe('testing aria');`

Reply:
Thanks, @nichia I just updated the test 👍